<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/트레이 아이콘.webp</normaloff>icon/트레이 아이콘.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 177, 208);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="lockButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 8pt &quot;HY중고딕&quot;;</string>
    </property>
    <property name="text">
     <string>잠금 해제</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/lock_open-black-18dp/2x/baseline_lock_open_black_18dp.png</normaloff>icon/lock_open-black-18dp/2x/baseline_lock_open_black_18dp.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>28</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 28pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Account Manager</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>태그</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 8pt &quot;HY중고딕&quot;;</string>
    </property>
    <property name="text">
     <string>새로고침</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/refresh-black-18dp/2x/baseline_refresh_black_18dp.png</normaloff>icon/refresh-black-18dp/2x/baseline_refresh_black_18dp.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 8pt &quot;HY중고딕&quot;;</string>
    </property>
    <property name="text">
     <string>계정 추가</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/add_circle_outline-black-18dp/2x/baseline_add_circle_outline_black_18dp.png</normaloff>icon/add_circle_outline-black-18dp/2x/baseline_add_circle_outline_black_18dp.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>즐겨찾기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePassword">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 8pt &quot;HY중고딕&quot;;</string>
    </property>
    <property name="text">
     <string>비밀번호 변경</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/settings-black-18dp/2x/baseline_settings_black_18dp.png</normaloff>icon/settings-black-18dp/2x/baseline_settings_black_18dp.png</iconset>
    </property>
   </widget>
   <widget class="Line" name="FirstLine_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>991</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="Favorites" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>481</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>7</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;HY중고딕&quot;;
</string>
    </property>
    <property name="text">
     <string>잠금 상태입니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>340</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>정보 확인</string>
    </property>
   </widget>
   <widget class="QWidget" name="TagWidget" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>481</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>링크</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>패스워드</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="Line" name="FirstLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>991</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="FirstLine_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>991</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 177, 208);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="SearchReturn" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>961</width>
      <height>171</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/search-black-18dp/2x/baseline_search_black_18dp.png</normaloff>icon/search-black-18dp/2x/baseline_search_black_18dp.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;HY중고딕&quot;;
background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string>도움말</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/help_outline-black-18dp/2x/baseline_help_outline_black_18dp.png</normaloff>icon/help_outline-black-18dp/2x/baseline_help_outline_black_18dp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY헤드라인M</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;HY헤드라인M&quot;;</string>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
